indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -0,0 +1,145 @@
+
+import React, {useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {Button, Icon, Checkbox} from 'semantic-ui-react';
+import {Column, Table, SortDirection, WindowScroller} from 'react-virtualized';
+import _ from 'lodash';
+import {
+  TooltipIfTruncated,
+  ManagementPageSubTitle,
+  ManagementPageBackButton,
+} from 'indico/react/components';
+import {useNumericParam} from 'indico/react/util/routing';
+import {Translate} from 'indico/react/i18n';
+import {EditableType} from '../../models';
+import StateIndicator from '../../editing/timeline/StateIndicator';
+import './EditableList.module.scss';
+
+// import list from './list'; will remove
+
+export default function EditableList() {
+  const list = []; // will remove
+  const eventId = useNumericParam('confId');
+  const {type} = useParams();
+  const [sortBy, setSortBy] = useState('id');
+  const [sortDirection, setSortDirection] = useState('ASC');
+  const [sortedList, setSortedList] = useState(list);
+  const title = {
+    [EditableType.paper]: Translate.string('List of papers'),
+    [EditableType.slides]: Translate.string('List of slides'),
+    [EditableType.poster]: Translate.string('List of posters'),
+  }[type];
+
+  // eslint-disable-next-line no-shadow
+  function _sortList({sortBy, sortDirection}) {
+    const newList = _.sortBy(list, [sortBy]);
+    if (sortDirection === SortDirection.DESC) {
+      newList.reverse();
+    }
+    return newList;
+  }
+
+  // eslint-disable-next-line no-shadow
+  function _sort({sortBy, sortDirection}) {
+    setSortBy(sortBy);
+    setSortDirection(sortDirection);
+    setSortedList(_sortList({sortBy, sortDirection}));
+  }
+
+  function _cellRenderer(dataKey, rowIndex) {
+    return (
+      <>
+        {dataKey === 'status' && <StateIndicator index={rowIndex} state="accepted" circular />}
+        <TooltipIfTruncated>
+          <div index={rowIndex} className="rowColumn_tooltip" role="gridcell">
+            {sortedList[rowIndex][dataKey]}
+          </div>
+        </TooltipIfTruncated>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <ManagementPageSubTitle title={title} />
+      <ManagementPageBackButton url={dashboardURL({confId: eventId})} />
+      <Button icon color="blue">
+        <Icon name="setting" />
+        <Translate>Customize List</Translate>
+      </Button>
+      <Button content={Translate.string('Assign')} />
+      <Button content={Translate.string('Unassign')} />
+      <Button content={Translate.string('Set status')} />
+      <Button content={Translate.string('Download all files')} />
+      <Button color="blue" floated="right" content={Translate.string('Assign to myself')} />
+      <div styleName="editable-list">
+        <WindowScroller>
+          {({height}) => (
+            <Table
+              autoHeight
+              width={1000}
+              styleName="table"
+              height={height}
```

Comment:
Is it needed if you also provide `autoHeight` prop?

Reply:
Yes, `height` is a required `prop`.